src/public/parts/shells/cabinet/public/index.html

```
@@ -100,7 +101,7 @@ <h3 class="font-bold text-lg" data-i18n="cabinet.properties"></h3>
 				<button id="propSave" type="button" class="btn btn-primary" data-i18n="cabinet.save" data-prop-field></button>
 			</div>
 		</div>
-		<form method="dialog" class="modal-backdrop"><button>close</button></form>
+		<form method="dialog" class="modal-backdrop"><button data-i18n="util.common.close">close</button></form>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)